appinfo/info.xml

```diff
@@ -71,14 +71,14 @@ Negative:
 
 Positive:
 * the software for training and inferencing of this model is open source
-* the trained model is freely available, and thus can be ran on-premises
+* the trained model is freely available, and thus can be ran on-premises 
 
 Negative:
 * the training data is not freely available, limiting the ability of external parties to check and correct for bias or optimise the model’s performance and CO2 usage.
 
 Learn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).
 ]]></description>
-    <version>1.0.13</version>
+    <version>1.0.14</version>
```

Comment:
If you add features, you should bump the minor version, so it'd be `1.1.0` (see https://semver.org)

Reply:
Reverted the version bump.